pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -660,6 +660,13 @@ public static boolean isProtocolVersion(Connection con, int version) {
     return false;
   }
 
+  public static boolean haveIntegerDateTimes(Connection con) {
+    if (con instanceof PgConnection) {
```

Comment:
Please add `if (con == null) { throw new NullPointerException("Connection is null"); }` kind of check to avoid silent failures.